zerver/lib/export.py

```diff
@@ -152,6 +152,7 @@ class MessagePartial(TypedDict):
     "zerver_realmplayground",
     "zerver_realmreactivationstatus",
     "zerver_realmuserdefault",
+    "zerver_realmsession",
```

Comment:
This should probably be declared under `NON_EXPORTED_TABLES` as a table we don't intend to export/import.

Reply:
what is the reason for not exporting it ? as `NON_EXPORTED_TABLES` states the reason for not doing so for every table.